docs/statistics.md

```diff
@@ -0,0 +1,561 @@
+20 bytes IPv4 + 8 bytes of UDP + 16 bytes SRT header. Receiver side.
+
+# Interval-based measurements
+
+These values can be reset by calling `srt_bstats(..., int clear)` with `clear = 1`. \
+This is helpful to get statistical measurements within a certain period, e.g. 1 second.
+
+## pktSent
+
+Same as `pktSentTotal`, but on the interval.
+
+## pktRecv
+
+Same as `pktRecvTotal`, but on the interval.
+
+## pktSndLoss
+
+Same as `pktSndLossTotal`, but on the interval.
+
+## pktRcvLoss
+
+Same as `pktRcvLossTotal`, but on the interval.
+
+## pktRetrans
+
+Same as `pktRetransTotal`, but on the interval.
+
+## pktRcvRetrans
+
+Same as `pktRcvRetransTotal`, but on the interval.
+
+## pktSentACK
+
+Same as `pktSentACKTotal`, but on the interval.
+
+## pktRecvACK
+
+Same as `pktRecvACKTotal`, but on the interval.
+
+## pktSentNAK
+
+Same as `pktSentNAKTotal`, but on the interval.
+
+## pktRecvNAK
+
+Same as `pktRecvNAKTotal`, but on the interval.
+
+## pktSndFilterExtra
+
+Same as `pktSndFilterExtraTotal`, but on the interval.
+
+Introduced in v1.4.0. Refer to [SRT Packet Filtering & FEC](packet-filtering-and-fec.md).
+
+## pktRcvFilterExtra
+
+Same as `pktRcvFilterExtraTotal`, but on the interval.
+
+Introduced in v1.4.0. Refer to [SRT Packet Filtering & FEC](packet-filtering-and-fec.md).
+
+## pktRcvFilterSupply
+
+Same as `pktRcvFilterSupplyTotal`, but on the interval.
+
+Introduced in v1.4.0. Refer to [SRT Packet Filtering & FEC](packet-filtering-and-fec.md).
+
+## pktRcvFilterLoss
+
+Same as `pktRcvFilterLossTotal`, but on the interval.
+
+Introduced in v1.4.0. Refer to [SRT Packet Filtering & FEC](packet-filtering-and-fec.md).
+
+## mbpsSendRate
+
+Sending rate in Mbps. Sender side.
+
+## mbpsRecvRate
+
+Receiving rate in Mbps. Receiver side.
+
+## usSndDuration
+
+Same as `usSndDurationTotal`, but measured on the interval.
+
+## pktReorderDistance
+
+`SRTO_LOSSMAXTTL` sets the maximum reorder tolerance value. The internal algorithm 
+checks the order of incoming packets and adjusts the tolerance based on the reorder 
+distance, but not to a value higher than the maximum.
+
+SRT starts from 0 tolerance. Once it receives the first 
+reordered packet, it increases the tolerance to the distance in the sequence
+discontinuity of the two packets. \
+After 10 consecutive original (not retransmitted) packets come in order, the reorder distance
+is decreased by 1 with every such a packet.
+
+For example, assume packets with the following sequence
+numbers are being received: \
+1, 2, 4, 3, 5, 7, 6, 10, 8, 9
+SRT starts from 0 tolerance. Receiving packet with sequence number 4 has a discontinuity
+equal to one packet. The loss is reported to the sender.
+With the next packet (sequence number 3) a reordering is detected. Reorder tolerance is increased to 1. \
+The next sequence discontinuity is detected when the packet with sequence number 7 is received.
+The current tolerance value is 1, which equals to the gap. No loss is reported. \
```

Comment:
The current tolerance value is 1, which is equal to the gap. No loss is reported. \